cmd/stack/deploy.go

```diff
@@ -64,8 +65,10 @@ func Deploy(ctx context.Context) *cobra.Command {
 			if err == nil {
 				log.Success("Stack '%s' successfully deployed", s.Name)
 			}
-			if err := stack.ListEndpoints(ctx, s, ""); err != nil {
-				return err
+			if os.Getenv(model.OktetoWithinDeployCommandContextEnvVar) != "" {
```

Comment:
Shouldn't be `os.Getenv(model.OktetoWithinDeployCommandContextEnvVar) == ""`? We want to list endpoints only in case of executing a standalone `okteto stack deploy`, right?